cmd/root.go

```diff
@@ -90,6 +96,9 @@ func init() {
 	rootCmd.Flags().String("primary-color", "#333", "primary color for text")
 	rootCmd.Flags().String("favicon", "", "path to favicon")
 	rootCmd.Flags().Duration("read-header-timeout", defaultReadHeaderTimeout, "timeout for reading the request headers")
+	rootCmd.Flags().String("provider", "aws", "cloud provider (aws or azure)")
+	rootCmd.Flags().StringVar(&resourceBlockName, "resource-block-name", "FRGAPAC", "default resource block name")
+	rootCmd.Flags().StringVar(&subscriptionID, "subscription-id", "", "Azure subscription ID (default is none)")
```

Comment:
Using a `StringVar` instead of just defining the flag bypasses Viper, which means these can only be defined on the CLI.